src/pages/tokens/[id].tsx

```diff
@@ -0,0 +1,203 @@
+  return (
+    <div className='flex flex-row flex-wrap items-center my-4'>
+      <TokenIcon className='h-8 w-8 mr-2' />
+      <h1 className='text-2xl font-semibold'>{token.name}</h1>
+    </div>
+  )
+}
+
+function ListRight ({ token }: { token: TokenData }): JSX.Element {
+  return (
+    <AdaptiveList>
+      <AdaptiveList.Row>
+        <AdaptiveList.Cell name='Decimal'>{token.decimal} Places</AdaptiveList.Cell>
+      </AdaptiveList.Row>
+      <AdaptiveList.Row>
+        <AdaptiveList.Cell name='Limit'>{token.limit}</AdaptiveList.Cell>
+      </AdaptiveList.Row>
+      <AdaptiveList.Row>
+        <AdaptiveList.Cell name='LPS'>{token.isLPS ? 'Yes' : 'No'}</AdaptiveList.Cell>
+      </AdaptiveList.Row>
+      <AdaptiveList.Row>
+        <AdaptiveList.Cell name='Tradable'>
+          {(() => {
+            if (token.tradeable) {
+              return (
+                <span className='flex flex-wrap items-center'>Yes <IoCheckmarkCircleOutline className='h-4 w-4 text-green-500 ml-1' />
+                </span>
+              )
+            }
+            return (
+              <span className='flex flex-wrap items-center'>No <IoAlertCircleOutline className='h-4 w-4 text-gray-500 ml-1' />
+              </span>
+            )
+          })()}
+        </AdaptiveList.Cell>
+      </AdaptiveList.Row>
+      <AdaptiveList.Row>
+        <AdaptiveList.Cell name='Finalized'>{token.finalized ? 'Yes' : 'No'}</AdaptiveList.Cell>
+      </AdaptiveList.Row>
+      <AdaptiveList.Row>
+        <AdaptiveList.Cell name='Destruction Height'>{token.destruction.height}</AdaptiveList.Cell>
+      </AdaptiveList.Row>
+      <AdaptiveList.Row>
+        <AdaptiveList.Cell name='Destruction TX' className='flex space-x-3 items-center'>
+          <div className='break-all'>{token.destruction.tx}</div>
+          <CopyButton text={token.destruction.tx} />
+        </AdaptiveList.Cell>
+      </AdaptiveList.Row>
+    </AdaptiveList>
+  )
+}
+
+function ListLeft ({ token }: { token: TokenData }): JSX.Element {
+  return (
+    <AdaptiveList>
+      <AdaptiveList.Row>
+        <AdaptiveList.Cell name='Category'>{token.isDAT ? 'DAT' : 'LPS'}</AdaptiveList.Cell>
+      </AdaptiveList.Row>
+      <AdaptiveList.Row>
+        <AdaptiveList.Cell name='Symbol'>{token.symbolKey}</AdaptiveList.Cell>
+      </AdaptiveList.Row>
+      <AdaptiveList.Row>
+        <AdaptiveList.Cell name='Net Supply'>{token.limit}</AdaptiveList.Cell>
+      </AdaptiveList.Row>
+      <AdaptiveList.Row>
+        <AdaptiveList.Cell name='Mintable'>
+          {(() => {
+            if (token.mintable) {
+              return (
+                <span className='flex flex-wrap items-center'>Yes <IoCheckmarkCircleOutline className='h-4 w-4 text-green-500 ml-1' /></span>
+              )
+            }
+            return (
+              <span className='flex flex-wrap items-center'>No <IoAlertCircleOutline className='h-4 w-4 text-gray-500 ml-1' /></span>
+            )
+          })()}
```

Comment:
This too can be done with a ternary.

Reply:
Made it like that for clarity purpose and to keep everything consistent. 

take a look at 

https://github.com/DeFiCh/scan/blob/8c647698a398586f644b0de329cbcc4185571b50/src/pages/tokens/index.tsx#L54-L63